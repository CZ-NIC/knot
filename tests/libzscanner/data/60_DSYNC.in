$ORIGIN	.
$TTL	1

; OK
@	DSYNC	CDS	NOTIFY	53	*._dsync.example.
@	TYPE66	CSYNC	1	5360	.
@	dsync	type59	0	0	\000\001\002\003\004\005\006\007\008\009

; KO
@	DSYNC
@	DSYNC	\# 0
@	DSYNC	CDS
@	DSYNC	JUNK	NOTIFY	53	example.com.
@	DSYNC	CDS	JUNK	53	junk._dsync.example.
@	DSYNC	CDS	NOTIFY	65567	port.number.overflow.
@	DSYNC	CDS	NOTIFY	5360	too-long-label.1234567890123456789012345678901234567890123456789012345678901234.example.com.
