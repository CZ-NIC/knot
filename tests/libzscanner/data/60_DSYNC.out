OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0042
RDATA=003B010035012A065F6473796E63076578616D706C6500
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0042
RDATA=003E0114F000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0042
RDATA=003B0000000A0001020304050607080900
------
WARNG=ZS_UNSUPPORTED_TYPE
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_UNSUPPORTED_TYPE
------
WARNG=ZS_UNSUPPORTED_TYPE
------
WARNG=ZS_BAD_DSYNC_SCHEME
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_LABEL_OVERFLOW
------
